code/datums/components/storage/storage.dm

```diff
@@ -870,3 +877,7 @@
 			user.balloon_alert(user, "[parent] now picks up all items")
 		if(COLLECT_ONE)
 			user.balloon_alert(user, "[parent] now picks up single item")
+
+/datum/component/storage/proc/animate_parent()
+	animate(parent, time = 1.5, loop = 0, transform = matrix().Scale(1.11, 0.85))
+	animate(time = 2, transform = null)
```

Comment:
this will remove item scaling, you should cache the original matrix and then re-assign it after 2 seconds

Reply:
That unfortunately doesn't seem to be possible, as I cannot fetch transform from parent.
However, the only feasible scenario where this will matter is adminbus. But in that case they can just set the animated variable to false.

Reply:
```
var/datum/parent = (some datum)
var/atom/A = parent
A.transform
```
Unfortunately byond doesn't support generic types so parent is a datum and not typed

Reply:
There, I made it work